tests/pkg/config/auditlog.go

```diff
@@ -1,9 +1,11 @@
 package config
 
 type AuditlogConfig struct {
-	TokenURL          string
-	ClientID          string
-	ClientSecret      string
```

Comment:
You removed the cliedSecret field but left the env var in the test chart, consider removing(or left them) in both places